src/Education/Commands/RuleDescriptionUserControl.xaml

```diff
@@ -130,7 +137,8 @@
             <Paragraph>By default, this tool window is docked with the
                 <Italic>Solution Explorer</Italic>. However, you can move it any where you like, just like any other tool window.
             </Paragraph>
-            <Paragraph>You can browse all available rules at <Hyperlink NavigateUri="https://rules.sonarsource.com/">https://rules.sonarsource.com</Hyperlink>
+            <Paragraph TextAlignment="Left">You can browse all available rules at
```

Comment:
Miscellaneous: changed the text alignment to `Left`. Before, it was being justified, which looked at bit weird depending on the width of the tool window.

Reply:
Ah good spot!